packages/headless/src/features/configuration/search-configuration-actions-loader.ts

```diff
@@ -1,6 +1,7 @@
 import {PayloadAction} from '@reduxjs/toolkit';
 import {RecommendationEngine} from '../../app/recommendation-engine/recommendation-engine';
-import {configuration, pipeline, searchHub} from '../../app/reducers';
+import {configuration} from '../../app/reducers';
+import {pipeline, searchHub} from '../../app/reducers';
```

Comment:
Did you mean to split those?

Suggestion to revert the change:
```suggestion
import {configuration, pipeline, searchHub} from '../../app/reducers';
```